build.gradle

```diff
@@ -296,3 +352,14 @@ project ('spring-pulsar-sample-apps') {
 	}
 }
 
+sonarqube {
```

Comment:
Nice addition! I saw this and the copyrights in spring-integration and was wanting to add it here as well. You beat me to it. :)